webview/src/experiments/components/table/styles.module.scss

```diff
@@ -325,6 +365,8 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
     }
 
     &.rowSelected {
+      border-color: $row-border-selected-color;
```

Comment:
Originally, we wanted to take the final border of multiple selected rows off, but I couldn't find a way to do that without writing some complex JS solution: 

<img width="800" alt="Screen Shot 2022-09-07 at 6 41 52 PM" src="https://user-images.githubusercontent.com/43496356/189001849-92638d65-05c8-436c-853e-d0409b1e5be3.png">
<img width="479" alt="Screen Shot 2022-09-07 at 6 42 03 PM" src="https://user-images.githubusercontent.com/43496356/189001850-2a79d03d-022c-41a2-ba9f-ef60b462e70f.png">

Reply:
Q: CSS :last-child selector? Or something similar?

Reply:
> Q: CSS :last-child selector? Or something similar?

A CSS selector doesn't work correctly due to how the table is built out of nested rows vs all rows being siblings.